.github/workflows/stale.yml

```diff
@@ -0,0 +1,43 @@
+name: close stale issues and pr
+on:
+  schedule:
+    - cron: "30 1 * * *"
+  workflow_dispatch:
+
+jobs:
+  close-issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - uses: actions/stale@v5
+        with:
+          debug-only: true
+          days-before-issue-stale: 28
+          days-before-issue-close: 7
+          any-of-labels:
+            [
+              "Waiting on author",
+              "Cannot reproduce 🤷",
+              "python past end of life",
+              "Won't fix/not planned",
+              "Work in progress",
+            ]
+          exempt-issue-labels: ["High priority", "Blocked 🚧"]
+          stale-issue-message:
+            "This issue is stale because it has been open 4 weeks with no activity.
+            Remove 'Stale' label or comment or this will be closed in a week."
+          close-issue-message:
+            "This issue was closed because it has been stalled for a week with no
+            activity."
+
+          days-before-pr-stale: 56
+          days-before-pr-close: 112
+          stale-pr-label: "Needs take over 🛎️"
```

Comment:
Can we exclude PRs with a label of needs-review? I wouldn't be comfortable marking PRs stale without an initial review.

Reply:
For me this label is really indicative of PRs which it would just be a shame to lose the time investment of. But if you really want to close these I'll give in.

Reply:
I just don't see how it's a loss, it's in the repository permanently, and easily searchable with the label we've already given it.

Reply:
Looks like Pierre is suggesting not closing PRs for the moment, at least while we try this out. See the suggestion above to use `-1` days.